main.go

```diff
@@ -90,8 +168,26 @@ func main() {
 
 		t := raw.Projects[i]
 
+		var url string
+		// check if it's a common git path `go get` supports and if not find real path
+		if !IsCommonPath(t.Name) {
+			realURL, err := FindRealPath(t.Name)
+
+			if err != nil {
+				//fmt.Printf("could not find real git url for import path %v: %+v\n", t.Name, err)
+				log.Fatal(err)
+			}
+			url = realURL
+		}
+
 		// special case: exception for golang.org/x based dependencies
-		url := "https://" + strings.Replace(t.Name, "golang.org/x/", "go.googlesource.com/", 1)
+		// if strings.Contains(t.Name, "golang.org/x/") {
+		// 	url = "https://" + strings.Replace(t.Name, "golang.org/x/", "go.googlesource.com/", 1)
+		// }
```

Comment:
I don't see any need to include this commented out code block to be included in your PR.